OSRMTextInstructionsTests/OSRMTextInstructionsTests.swift

```diff
@@ -101,10 +107,14 @@ class OSRMTextInstructionsTests: XCTestCase {
         // needs, but that our not in the fixtures
         var fixture: [String: Any] = [:]
         var maneuver: [String: Any] = [
-            "location": [ 1.0, 1.0 ]
+            "location": [ 1.0, 1.0 ],
+            "distance": 0,
         ]
         var step: [String: Any] = [
-            "mode": "driving"
+            "mode": "driving",
+            "driving_side": "right",
+            "distance": 0,
+            "duration": 0,
```

Comment:
These fixture changes are likely necessary either way since they're likely a product of the MapboxDirections upgrade adding fields that are required in the response(s).